examples/config.json

```diff
@@ -2,36 +2,22 @@
         "basic_auth_username": "username",
         "basic_auth_password": "password",
         "log_level": "info",
-        "secret": "reverse-pendulum",
-        "aws_config": {
-                "region": "eu-west-1"
-        },
+        "cloud": "aws-eu-west-1",
         "catalog": {
                 "services": [{
                         "id": "uuid-1",
-                        "name": "mongodb",
-                        "description": "MongoDB clusters via AWS CloudFormation",
+                        "name": "elasticsearch",
+                        "description": "Elasticsearch instances provisioned via Aiven",
                         "bindable": true,
                         "requires": [],
                         "metadata": {},
-                        "bastion_security_group_id": "sg-xxxxxx",
-                        "key_pair_name": "key_pair_name",
-                        "vpc_id": "vpc-xxxxxx",
-                        "primary_node_subnet_id": "subnet-xxxxxx",
-                        "secondary_0_node_subnet_id": "subnet-xxxxxx",
-                        "secondary_1_node_subnet_id": "subnet-xxxxxx",
                         "plans": [{
                                 "id": "uuid-2",
                                 "name": "basic",
-                                "description": "No replicas. Disk: 400GB gp2. Instance: m4.large",
-                                "metadata": {},
-                                "cluster_replica_set_count": "1"
-                        },{
-                                "id": "uuid-2",
-                                "name": "replica-set-3",
-                                "description": "A replica set of 3 instances. Disk: 400GB gp2. Instance: m4.large",
-                                "metadata": {},
-                                "cluster_replica_set_count": "3"
+                                "aiven_plan": "startup-1",
+                                "elasticsearch_version": "6",
```

Comment:
We talked at kickoff abour keeping the plan config data out of the catalog, and following the pattern used by the elasticache broker. I don't see that being done here...

Reply:
I agree with Alex as now there is duplication in the broker config and the provider config.

Reply:
The fields mentioned here aren't rendered into the broker config. The same config JSON gets unmarshalled into two different data structures: one for the broker and one for the provider. The separation you asked for is there, its just the same source file gets used.

This duplication is on purpose and came without development effort by using another library as a starting point. It would be more work to remove it.